src/lib_json/json_reader.cpp

```diff
@@ -18,6 +18,8 @@
 #include <memory>
 #include <set>
 #include <limits>
+#include <stdio.h>
```

Comment:
Beats me. I dont use modern compilers or know anything about them. This was failing on visual c++ 2010.